http-server-netty/src/test/groovy/io/micronaut/http/server/netty/binding/ByteBufferSpec.groovy

```diff
@@ -32,6 +33,7 @@ class ByteBufferSpec extends Specification {
 
     @Inject
     @Client("/")
+    @AutoCleanup
```

Comment:
should we recommend always adding `@AutoCleanup` when injecting an `HttpClient` in tests to close it?  We are not doing that. E.g. tests here: https://guides.micronaut.io/latest/micronaut-http-client-gradle-groovy.html#tests

Reply:
it isnt really a problem unless you do leak detection

Reply:
but we should do it? it does not cost us anything to change the best practices in guides.

Reply:
sure

Reply:
`@AutoCleanup` is not needed here the client is shutdown when the context is stopped